site/docs/index.md

```diff
@@ -5,62 +5,34 @@ title: Home
 
 
 
+## What is Substrait?
 
+Substrait is a format for describing compute operations on structured data. It is designed for interoperability across different languages and systems.
 
-## Project Vision
 
-Create a well-defined, cross-language [specification](spec/specification) for data compute operations. This includes a declaration of common operations, custom operations and one or more serialized representations of this specification. The spec focuses on the semantics of each operation and a consistent way to describe.
 
-In many ways, the goal of this project is similar to that of the Apache Arrow project. Arrow is focused on a standardized memory representation of columnar data. Substrait is focused on what should be done to data.
+## How does it work?
 
+Substrait provides a well-defined, cross-language [specification](spec/specification) for data compute operations.  This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.  The spec focuses on the semantics of each operation.
```

Comment:
I'd an additional sentence here along the lines of "In addition the format and specification, the Substrait community also includes a number of libraries and other useful tools [to get started](/do_we_have_a_list_of_tools_like_validator_isthmus_etc)

Reply:
Done.